packages/flutter/test/widgets/overscroll_stretch_indicator_test.dart

```diff
@@ -610,15 +610,28 @@ void main() {
 
     final TestGesture pointer2 = await tester.startGesture(tester.getCenter(find.text('Index 1')));
     // Add overscroll from an additional pointer
-    await pointer2.moveBy(const Offset(0.0, 210.0));
+    // Here are two pointers, so when pointer2 moved, the gesture will wait (pointer1's move) or (pointer2's move).
+    await pointer2.moveBy(const Offset(0.0, 105.0));
+    await tester.pumpAndSettle();
+    expect(find.text('Index 1'), findsOneWidget);
+    expect(tester.getCenter(find.text('Index 1')).dy, lastStretchedLocation);
```

Comment:
If only one pointer down and move. The gesture will update immediately when we move the pointer by 105. But
 if there are 2 pointers down and only one pointer moves by 105, there is no change to scroll because a batch is not completed.